ToggleButton/ToggleButton.module.less

```diff
@@ -70,11 +74,30 @@
 			color: @agate-togglebutton-color;
 		}
 
+		.toggleIndicator {
+			color: @agate-togglebutton-indicator-off-color;
+			font-size: @agate-togglebutton-indicator-font-size;
+			margin: @agate-togglebutton-indicator-margin;
+		}
+
+		&.minWidth {
+			&,
+			&.huge,
+			&.small,
+			&.smallest {
+				padding: 0 @agate-togglebutton-h-padding;
```

Comment:
This will affect the ToggleButton in other skins.

Reply:
I will remove this